superdesk/publish/formatters/email_formatter.py

```diff
@@ -46,13 +46,14 @@ def format(self, article, subscriber, codes=None):
         pub_seq_num = superdesk.get_resource_service('subscribers').generate_sequence_number(subscriber)
         doc = {}
         try:
-            # If there is a dateline inject it into the body
-            if formatted_article.get(FORMAT) == FORMATS.HTML and formatted_article.get('dateline', {}).get('text'):
-                body_html_elem = sd_etree.parse_html(formatted_article.get('body_html'))
-                ptag = body_html_elem.find('.//p')
-                if ptag is not None:
-                    ptag.text = formatted_article['dateline']['text'] + ' ' + (ptag.text or '')
-                    formatted_article['body_html'] = sd_etree.to_string(body_html_elem)
+            if formatted_article.get(FORMAT) == FORMATS.HTML:
+                # If there is a dateline inject it into the body
+                if formatted_article.get('dateline', {}).get('text'):
```

Comment:
imo this could use a function, otherwise it's hard to see what that change was, there is too much going on